src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/ResourcesRoutes.scala

```diff
@@ -119,7 +119,7 @@ class ResourcesRoutes(resources: Resources[Task])(implicit cache: DistributedCac
 
     def search(implicit acls: FullAccessControlList, caller: Caller, wrapped: LabeledProject) =
       (pathPrefix(IdSegment / "sparql") & post & entity(as[String]) & pathEndOrSingleSlash) { (id, query) =>
-        (identity(caller) & hasPermission(resourceRead)) { implicit ident =>
+        (identity(caller) & hasPermission(resourceRead)) { _ =>
```

Comment:
is the identity still propagated if you remove the implicit ?

Reply:
The identity is not used here, hence the compiler warning.